travis.sh

```diff
@@ -64,6 +64,12 @@ function error {
     exit 1
 }
 
+# Start prerelease, and once it finishs then finish this script too.
+if [ "$PRERELEASE" == true ]; then
+  ./ros_pre-release.sh;
```

Comment:
This does not work, I would suggest to use the same path composition as for rosdep-install.sh